daffodil-lib/src/main/resources/org/apache/daffodil/xsd/dafint.xsd

```diff
@@ -17,7 +17,7 @@
 -->
 
 <xsd:schema xmlns:xsd="http://www.w3.org/2001/XMLSchema"
-  attributeFormDefault="unqualified" elementFormDefault="qualified"
+  elementFormDefault="qualified"
   targetNamespace="urn:ogf:dfdl:2013:imp:daffodil.apache.org:2018:int"
```

Comment:
I think for all these normal schemas (i.e. not dfdl.xsd's) we probably do want the attributeFormatDefault property. That property is valid and is used when doing normal schema validation. It's only ignore in DFDL schemas since the property isn't part of the DFDL specification.

Reply:
Ok I will take out the change for removing attributeFormDefault in these schemas.